website/static/js/fangorn.js

```diff
@@ -49,6 +49,64 @@ var getExtensionIconClass = function(name) {
     return null;
 };
 
+function findByTempID(parent, tmpID){
+    var child;
+    var item;
+    for (var i = 0; i < parent.children.length; i++) {
+        child = parent.children[i];
+        if (!child.data.tmpID) {
+            continue;
+        }
+        if (child.data.tmpID === tmpID) {
+            item = child;
+        }
+    }
+    return item;
+}
+
+function cancelUploads (row) {
+    var treebeard = this;
+    var filesArr = treebeard.dropzone.getQueuedFiles();
+    for (var i = 0; i < filesArr.length; i++) {
+        var m = filesArr[i];
```

Comment:
Prefer more descriptive variable names. How about renaming `filesArr` to `files` and `m` to `file`, or something like that?